capa/features/common.py

```diff
@@ -405,6 +405,12 @@ def __init__(self, value: str, description=None):
         self.name = "os"
 
 
+class ScriptLanguage(Feature):
+    def __init__(self, value: str, description=None):
+        super().__init__(value, description=description)
+        self.name = "script language"
```

Comment:
could we use `format` for this? e.g. `format: C#`.

pro:
  - fewer features to memorize
  - less duplication
  - less code

con:
  - maybe slightly less precise